trigger/kafkasub/trigger.go

```diff
@@ -210,111 +216,90 @@ func initKafkaParms(t *KafkaSubTrigger) error {
 		}
 		t.kafkaParms.brokers[brokerNo] = broker
 	}
+	//clientKeystore
+	/*
+		Its worth mentioning here that when the keystore for kafka is created it must support RSA keys via
+		the -keyalg RSA option.  If not then there will be ZERO overlap in supported cipher suites with java.
+		see:   https://issues.apache.org/jira/browse/KAFKA-3647
+		for more info
+	*/
+	if t.config.Settings["truststore"] != nil {
```

Comment:
No need to change it, but have a look at "The expression may be preceded by a simple statement" 
in https://golang.org/ref/spec#If_statements, it is a cool feature :)